lib/remote/configobjectutility.cpp

```diff
@@ -112,9 +118,24 @@ bool ConfigObjectUtility::CreateObject(const Type::Ptr& type, const String& full
 	String path = GetObjectConfigPath(type, fullName);
 	Utility::MkDirP(Utility::DirName(path), 0700);
 
-	if (Utility::PathExists(path)) {
-		errors->Add("Configuration file '" + path + "' already exists.");
-		return false;
+	{
+
+#ifdef _WIN32
+#define stat _stat
+#endif /* _WIN32 */
```

Comment:
That's bad coding practice to rename functions. Please remove that and change the underlaying code.